### ISSUE TITLE
Unusual docstring format

### ISSUE THREAD

**Issue by gdude2002** [prodicus/vocabulary#6]
While this isn't really a huge problem, you may wish to reformat your docstrings according to the [established formats](http://stackoverflow.com/a/24385103/4022104).

[PEP 287](https://www.python.org/dev/peps/pep-0287) recommends that reST (reStructuredText) format be used, but Epytext is popular as well - They're used because Python IDEs are often able to parse them to get more information about the things they document, for example, for auto-completion.

**Comment by prodicus** [comment]
Didn't know about this. Will be refactoring the docstrings to conform with [PEP 287](https://www.python.org/dev/peps/pep-0287). reST seems like an obvious choice here. 

Thanks for pointing out :smile:

**Comment by gdude2002** [comment]
:+1: - it's not a big deal, but it's one of those useful little things that gets overlooked a lot.

**Comment by prodicus** [comment]
Fixed in  295ee0b. 

Closing issue.

**Comment by balta2ar** [comment]
`:param` and `:returns` are great but I also suggest adding `:type` and `:rtype` fields. That helps **A LOT** in the long run.

**Comment by prodicus** [comment]
An example if you don't mind @balta2ar ? Would surely add it up

**Comment by balta2ar** [comment]
Something like this:
``` python
    @defer.inlineCallbacks
    def _update_models(self, cur_time,
                       new_server_stats, max_time_server):
        """
        Updates global models and models of budget objects.

        @param cur_time: Current time
        @type cur_time: datetime.datetime

        @param new_server_stats: New server stats
                (received since last update).
        @type new_server_stats: ProcessedServerStats

        @param max_time_server: Timestamp (unix timestamp) of latest
                update got from server[s].
        @type max_time_server: float

        @return: Update models running deferred.
        @rtype: defer.Deferred(dict[str, ComputationResult])
        """
```

**Comment by gdude2002** [comment]
Yuri, that isn't rst. It would be more like this:

* `:type param: str` - Expected type of a parameter
* `:rtype: bool` - Type/s of object/s returned by the function

**Comment by balta2ar** [comment]
@gdude2002 Absolutely, that was just an example of `rtype` that I've grabbed from some project.
